cli/src/main.ts

```diff
@@ -0,0 +1,11 @@
+import { enableProdMode } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import { AppModule } from './app/app.module';
+import { environment } from './environments/environment';
+
+if (environment.production) {
+  enableProdMode();
+}
+
+platformBrowserDynamic().bootstrapModule(AppModule);
```

Comment:
you need document ready

Reply:
Not sure what you mean here. This the the boilerplate bootstrap config generated by the CLI's current template

Reply:
Ideally you really do want to wait until document ready, it does stop a lot of bizarre random edge cases from popping up.
This should probably be baked into the CLI as well.

```typescript
document.addEventListener('DOMContentLoaded', () => {
   platformBrowserDynamic().bootstrapModule(AppModule);
});
```